go.mod

```diff
@@ -1,104 +1,106 @@
 module github.com/openshift/sdn
 
-go 1.18
+go 1.20
 
 require (
 	github.com/containernetworking/cni v0.8.1
 	github.com/containernetworking/plugins v0.6.0
 	github.com/fsnotify/fsnotify v1.6.0
 	github.com/gorilla/mux v1.8.0
 	github.com/miekg/dns v1.1.35
-	github.com/opencontainers/runc v1.1.4
-	github.com/openshift/api v0.0.0-20220613185440-63b58097a95d
-	github.com/openshift/build-machinery-go v0.0.0-20211213093930-7e33a7eb4ce3
-	github.com/openshift/client-go v0.0.0-20220603133046-984ee5ebedcf
-	github.com/openshift/library-go v0.0.0-20220525173854-9b950a41acdc
-	github.com/prometheus/client_golang v1.14.0
-	github.com/spf13/cobra v1.6.0
+	github.com/opencontainers/runc v1.1.7
+	github.com/openshift/api v0.0.0-20231010075512-1ccc6058c62d
+	github.com/openshift/build-machinery-go v0.0.0-20220913142420-e25cf57ea46d
+	github.com/openshift/client-go v0.0.0-20230926161409-848405da69e1
+	github.com/openshift/library-go v0.0.0-20231101160200-9e6e65ed2a52
+	github.com/prometheus/client_golang v1.16.0
+	github.com/spf13/cobra v1.7.0
 	github.com/vishvananda/netlink v1.1.0
 	go.opentelemetry.io/otel/trace v1.10.0
-	golang.org/x/sys v0.11.0
-	k8s.io/api v1.27.1
-	k8s.io/apimachinery v1.27.1
-	k8s.io/apiserver v1.27.1
-	k8s.io/client-go v1.27.1
+	golang.org/x/sys v0.13.0
+	k8s.io/api v1.28.3
+	k8s.io/apimachinery v1.28.3
+	k8s.io/apiserver v1.28.3
+	k8s.io/client-go v1.28.3
 	k8s.io/cloud-provider v0.0.0
-	k8s.io/component-base v1.27.1
+	k8s.io/component-base v1.28.3
 	k8s.io/component-helpers v0.0.0
-	k8s.io/cri-api v1.27.1
-	k8s.io/klog/v2 v2.90.1
-	k8s.io/kubernetes v1.27.1 // actually branch sdn-4.14-kubernetes-1.27.1 in openshift/kubernetes
-	k8s.io/utils v0.0.0-20230209194617-a36077c30491
+	k8s.io/cri-api v1.28.3
+	k8s.io/klog/v2 v2.100.1
+	k8s.io/kubernetes v1.28.3 // actual branch sdn-4.15-kubernetes-1.28.3 in openshift/kubernetes
+	k8s.io/utils v0.0.0-20230726121419-3b25d923346b
 	sigs.k8s.io/yaml v1.3.0
 )
 
+require github.com/spf13/pflag v1.0.5
```

Comment:
you ought to be able to put that above with all the others

Reply:
done